glide.yaml

```diff
@@ -1,5 +1,13 @@
 package: github.com/giantswarm/k8scloudconfig
 import:
+- package: github.com/giantswarm/apiextensions
+  version: clientset
+- package: k8s.io/apiextensions-apiserver
+  version: release-1.7
+- package: k8s.io/apimachinery
+  version: release-1.7
+- package: k8s.io/client-go
+  version: v4.0.0
```

Comment:
This has to be pinned like we do in operatorkit because of the Kubernetes magic with interfaces and code generation. Once we are running on CRDs I will remove this.